links.md

```diff
@@ -1,4 +1,27 @@
 [infomail]: mailto:socratesuk-info@googlegroups.com
 [twitter]: https://twitter.com/socrates_uk
 [softwarecrafters]: https://www.softwarecrafters.org/
+[LSCC]: http://www.meetup.com/london-software-craftsmanship/
+
+[WottonHouse]: http://www.wottonhousedorkingsurrey.co.uk/
+[WottonGoogleMaps]: https://www.google.co.uk/maps/place/Wotton+House/@51.210842,-0.3960782,135m/data=!3m1!1e3!4m7!1m4!3m3!1s0x0:0x0!2zNTHCsDEyJzM5LjIiTiAwwrAyMyc0NC42Ilc!3b1!3m1!1s0x0000000000000000:0x9b7aa884271af6b5!6m1!1e1?hl=en
+
+[Details]: details.html
+
+[Codurance]: http://www.codurance.com
+
+[SoCraTes Germany]: http://www.socrates-conference.de/
 [socrates_de_values]: https://www.socrates-conference.de/values
+[Farncombe Gallery]: http://farncombeconferencecentre.com/venue/gallery/
+[Conference Code of Conduct]: http://confcodeofconduct.com/
+[CodeFreeze]: http://www.codefreeze.fi/
+[SoCraTes Canaries]: https://socracan.com/
+[I T.A.K.E. Unconference]: http://itakeunconf.com/
```

Comment:
I'm all for removing duplication and I love how this change cleared up some.

Moving these conference links somewhere else than where they're used might make it harder for someone to add/update one of them. Same goes for sponsorship links and maybe others that are only connected to a specific part of the page. Just a thought!

Reply:
I agree, that it means you need to look into 2 files.
I personally prefer to have a consistent way of working though, if we have most links in one file and some of them where they are used I fear people will start clutter them all over the place again

Reply:
yes, that's the tradeoff. I don't have a strong feeling about it.